### PR TITLE
Turn order

### PR DESCRIPTION
**Pull request by rzulian**
Revised turn order with rounds. Now also faction selection and initial buildings are manage with turn order

### CODE REVIEW

**Comment by rzulian** [comment]
@coyotte508 can you check now?

File: src/faction-boards/types.ts [hunk 1]
```diff
@@ -19,7 +19,11 @@ export interface FactionBoardRaw {
   };
   [Building.Academy1]?: {
     cost?: string,
-    income?: string[]
+    income?: string
+  };
+  academy2?: {
```
Comment (coyotte508) [inline src/faction-boards/types.ts]: This should be removed. There's `[Building.academy2]` below

File: src/faction-boards/types.ts [hunk 2]
```diff
@@ -85,7 +89,11 @@ export class FactionBoard {
   };
   [Building.Academy1]: {
     cost: Reward[],
-    income: Event[]
+    income: Event
+  };
+  academy2: {
```
Comment (coyotte508) [inline src/faction-boards/types.ts]: This should be removed too

**Comment by rzulian** [comment]
should be fine now

**Pull request merged**